app/cinema/hall.py

```diff
@@ -0,0 +1,19 @@
+from app.people.customer import Customer
+from app.people.cinema_staff import Cleaner
+
+
+class CinemaHall:
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(
+            self,
+            movie_name: str,
+            customers: list,
+            cleaning_staff: Cleaner
+    ) -> None:
+        print(f'"{movie_name}" started in hall number {self.number}.')
+        for people in customers:
+            Customer.watch_movie(self=people, movie=movie_name)
+        print(f'"{movie_name}" ended.')
+        Cleaner.clean_hall(cleaning_staff, self.number)
```

Comment:
```suggestion
            customer.watch_movie(movie_name)
        print(f'"{movie_name}" ended.')
        cleaning_staff.clean_hall(self.number)
```

Call the methods like these